Language/Haskell/GhcMod/CabalHelper.hs

```diff
@@ -132,6 +134,16 @@ getCustomPkgDbStack = do
     mCusPkgDbFile <- liftIO . (traverse readFile <=< findCustomPackageDbFile) . cradleRootDir =<< cradle
     return $ parseCustomPackageDb <$> mCusPkgDbFile
 
+getStackPackageDbStack :: IOish m => m [GhcPkgDb]
+getStackPackageDbStack = do
+    mstack <- liftIO $ findExecutable "stack"
+    case mstack of
+      Nothing -> return []
+      Just stack -> do
+        snapshotDb <- liftIO $ readProcess stack ["path", "--snapshot-pkg-db"] ""
```

Comment:
speaking of stdout/stderr, if stack is likely to print stuff on stdout or sterr in the future we should use `gmReadProcess` here and wherever else we invoke that exe.